android/AndroidManifest.xml

```diff
@@ -66,12 +63,14 @@
       android:label="@string/app_name"
       android:localeConfig="@xml/locales_config"
       android:theme="@style/MwmTheme"
+      android:resizeableActivity="true"
```

Comment:
What is the impact of it for normal phones?

Reply:
Setting this attribute to "true", allows the user can launch the activity in split-screen and free-form modes. If you set the attribute to ”false”, the app can't be tested or optimized for a multi-window environment.

Reply:
Are there any additional effects from enabling it? How OpenGL/Vulkan windows behave after it is enabled? Are there any additional callbacks or other changes in the system behavior?

Reply:
I couldn't find any additional side effects from enabling it. Everything should behave as the same given that this only allows optional resizing and does not change anything else. Theoretically, it could cause the startup time to be higher if launched on two, three, four etc. displays on the same time but the user must explicitly trigger this.

There are the new activities created:
> When you launch a new activity, you can indicate that the new activity should be displayed adjacent to the current one if possible. Use the intent flag [FLAG_ACTIVITY_LAUNCH_ADJACENT](https://developer.android.com/reference/android/content/Intent#FLAG_ACTIVITY_LAUNCH_ADJACENT), which tells the system to try to create the new activity in an adjacent window, so the two activities share the screen. The system makes a best effort to do this, but it is not guaranteed to happen.

If a device is in free-form mode and you are launching a new activity, you can specify the new activity's dimensions and screen location by calling [ActivityOptions.setLaunchBounds()](https://developer.android.com/reference/android/app/ActivityOptions#setLaunchBounds(android.graphics.Rect)). This method has no effect if the device is not in multi-window mode.

Reply:
Did you test this resizing on real devices?